src/main/java/com/icthh/xm/uaa/domain/properties/TenantProperties.java

```diff
@@ -38,6 +38,7 @@
      */
     @JsonProperty("strictUserManagement")
     private boolean strictUserManagement;
+    private boolean multiRoleEnabled;
```

Comment:
add JsonProperty and enter for keep same style

Reply:
suggest moving to Security container